stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListViewStyle.kt

```diff
@@ -84,6 +87,8 @@ import io.getstream.chat.android.ui.message.list.internal.ScrollButtonView
 public data class MessageListViewStyle(
     public val scrollButtonViewStyle: ScrollButtonViewStyle,
     public val scrollButtonBehaviour: MessageListView.NewMessagesBehaviour,
+    public val scrollButtonBottomMargin: Int,
+    public val scrollButtonEndMargin: Int,
```

Comment:
I know grouping the items is good, but shouldn't you add the parameters to the bottom, so nothing brakes due to the parameter ordering?